src/citra_qt/main.cpp

```diff
@@ -1410,6 +1416,28 @@ void GMainWindow::OnRemoveAmiibo() {
     ui.action_Remove_Amiibo->setEnabled(false);
 }
 
+void GMainWindow::OnMenuSelectEmulatedDirectory(const FileUtil::UserPath path) {
+    const auto res = QMessageBox::information(this, tr("Changing Emulated Directory"),
+                                              tr("You are about to change the emulated %1 directory of the system. Please note "
+                                                 "that this does not also move the contents of the previous directory to the "
```

Comment:
For some reason, this line is somewhat confusing to me. I think it would be better to write:

```suggestion
                                                 "that this will NOT move the contents of the previous directory to the "
```